tech-far-hub/package.json

```diff
@@ -41,6 +41,7 @@
     "react-icons": "^4.7.1",
     "react-use": "^17.4.0",
     "remark-gfm": "^1.0.0",
+    "remark-html": "^15.0.1",
```

Comment:
had to look this one up. Where is this used? I'm sure I'll get it to it as I progress down the file changes.

Reply:
I needed it for rendering the frontmatter as markdown.